config/initializers/exceptions.rb

```diff
@@ -1 +1,3 @@
 class RemovingLastTeamAdminException < RuntimeError; end
+
+class RemovingLastTeamException < RuntimeError; end
```

Comment:
Doesn't need to be a holdup on this PR, but it seems like these errors should live in the `bullet_train` gem since that's where they're raised and rescued.